src/controllers/red-experience.js

```diff
@@ -0,0 +1,74 @@
+import {ReturnState} from './_base.js';
+
+/**
+ * Clean the incoming POST request body to make it more compatible with the
+ * database and its validation rules.
+ *
+ * @param {object} body The incoming request's body.
+ * @returns {any} A json object that's just got our cleaned up fields on it.
+ */
+const cleanInput = (body) => {
+  return {
+    // Ensure the user has entered integers
+    cleanRedExperienceYears: body.redExperienceYears === undefined ? undefined : body.redExperienceYears.trim(),
+    cleanRedControlNumber: body.redControlNumber === undefined ? undefined : body.redControlNumber.trim()
```

Comment:
Either...
1. That comment is wrong.
2. You're not using the [util function we wrote to clean integers](https://github.com/Scottish-Natural-Heritage/fit-and-competent/blob/develop/src/controllers/_util.js#L8).